packages/blade/src/components/Carousel/Carousel.web.tsx

```diff
@@ -43,6 +43,7 @@ type ControlsProp = Required<
   onIndicatorButtonClick: (index: number) => void;
   onNextButtonClick: () => void;
   onPreviousButtonClick: () => void;
+  shouldShowNavigationButton: boolean;
```

Comment:
```suggestion
  showNavigationButtons: boolean;
```

similar to the prop we already have: `showIndicators`